containers/daap-get-schema/src/var/task/get_schema.py

```diff
@@ -0,0 +1,38 @@
+import json
+import os
+
+import botocore
+from data_platform_api_responses import format_response_json, response_status_404
+from data_platform_logging import DataPlatformLogger
+from data_platform_paths import DataProductConfig
+from dataengineeringutils3.s3 import read_json_from_s3
+
+logger = DataPlatformLogger(
```

Comment:
Could this be moved inside the handler?
Then it can be instantiated including the `"data_product_name": data_product_name` and `"table_name": table_name`. That means the log files created will have this information on every row that's logged